src/containers/NFT/NFTTabs/Offers.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useInfiniteQuery } from 'react-query'
+import { Link } from 'react-router-dom'
+import Loader from '../../shared/components/Loader'
+import './styles.scss'
+import noInfo from '../../shared/images/no_info.svg'
+import SocketContext from '../../shared/SocketContext'
+import { Amount } from '../../shared/components/Amount'
+import { analytics, ANALYTIC_TYPES } from '../../shared/utils'
+import '../../shared/components/TransactionTable/styles.scss' // Reuse load-more-btn
+
+const formatAmount = require('../../../rippled/lib/txSummary/formatAmount')
```

Comment:
why do we `require` instead of `import`?

Reply:
Fixed.